applications/oneramp.md

```diff
@@ -0,0 +1,170 @@
+| **0b.** | Documentation                                                          | We will provide both **inline documentation** of the product and a tutorial explanation of the product and how to use it.                                                                                              |
+| **0c.** | Testing and Testing Guide                                              | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness.                                                                                                               |
+| **0d.** | Docker                                                                 | We will provide a Dockercontainer that can be used to test all the functionality delivered with this milestone.                                                                                                        |
+|     0e. | Article                                                                | We will publish an **article**/workshop that explains the product and how it's being used in sub-saharan africa                                                                                                        |
+|      1. | Finish [oneramp polkadot app ](https://polkadot.oneramp.io)            | we will complete the oneramp polkadot app that will have on and offramping functionality in Uganda, Kenya, Ghana, Nigeria, Tanzania                                                                                    |
+|      2. | Documentation for intergation of oneramp for polkadot wallet providers | the most commonn form of on and offramp provider intergation is through re-direct links, We shall provide neccessary documentation to allow for wallet providers to onboard oneramp as a service in their applications |
+|      3. | Support for Malawi and Cameroon                                        | add support for Malawi and Cameroon allowing users in those regions to be able to on and offramp on polkadot through oneramp,                                                                                          |
+|      4. | Support for South Africa                                               | Add support for South Africa, This is the second-biggest crypto adoption market in sub-saharan Africa after Nigeria as per [chainalysis](https://www.chainalysis.com/blog/africa-cryptocurrency-adoption/)             |
+|      5. | marketing gampaign to create awareness about polkadot on oneramp       | Run an marketing gampaign to create awareness about polkadot on oneramp and how users can easily onboard                                                                                                               |
+
+### Milestone 2 Example — Additional features
```

Comment:
```suggestion
### Milestone 2 — Additional features
```

Reply:
great, on it. 
made the neccessary changes.